src/renderer/src/i18n/en-US/settings.json

```
@@ -771,8 +365,138 @@
+      "npmRegistry": {
+        "title": "NPM Registry Configuration",
+        "currentSource": "Current Source",
+        "cached": "Cached",
+        "lastChecked": "Last Checked",
+        "refresh": "Refresh",
+        "advanced": "Advanced",
+        "advancedSettings": "Advanced Settings",
+        "advancedSettingsDesc": "Configure advanced NPM registry options, including auto-detection and custom source settings.",
+        "autoDetect": "Auto-detect optimal source",
+        "autoDetectDesc": "Automatically detect and use the fastest NPM registry on startup.",
+        "autoDetectEnabled": "Auto-detect optimal NPM registry enabled",
```

Biome (2.1.2)

[error] The key autoDetectEnabled was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)